ClientApp/src/NavBar/NavBar.tsx

```diff
@@ -47,6 +47,7 @@ function NavBar() {
                         <a href="/getting-started" className={selectedPage == 'started' ? 'active' : 'inactive'} onClick={() => setSelectedPage('started')}>Getting Started</a>
                         <a href="/submit-sighting" className={selectedPage == 'submit-sighting' ? 'active' : 'inactive'} onClick={() => setSelectedPage('submit-sighting')}>Submit a Sighting</a>
                         <a href="/conservation" className={selectedPage == 'conservation' ? 'active' : 'inactive'} onClick={() => setSelectedPage('conservation')} >Conservation</a>
+                        <a href="/whale-species" className={selectedPage == 'whale-species' ? 'active' : 'inactive'} onClick={() => setSelectedPage('whale-species')} >Whale Species</a>
```

Comment:
It's probably to do with the styling of the NavBar but I checked out your branch and can't actually see this link in the nav bar at the moment!

Reply:
mariam, we have added links to your page on our navbar PR so you can just undo this! :)

Reply:
If this doesn't get merged in time do we need to undo the above?